app/scss/main.scss

```diff
@@ -1,6 +1,6 @@
 body,
 html {
-    font-family: proxima-nova, "Helvetica Neue", Helvetica, Arial, sans-serif;
+    font-family: "proxima-nova", "Helvetica Neue", Helvetica, Arial, sans-serif;
```

Comment:
we should prob dl a font library

Sent from my iPhone

> On Jul 2, 2016, at 10:10 AM, batmantis <notifications@github.com> wrote:
> 
> In app/scss/main.scss:
> 
> > @@ -1,6 +1,6 @@
> >  body,
> >  html {
> > -    font-family: proxima-nova, "Helvetica Neue", Helvetica, Arial, sans-serif;
> > +    font-family: "proxima-nova", "Helvetica Neue", Helvetica, Arial, sans-serif;
> I think we can include 'proxima-nova' font in our app, so that everyone will have it.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>